js/reducers/createReducer.js

```diff
@@ -0,0 +1,39 @@
+const initiellState = { data: {} };
+
+export const createReducer = (feilActionType,
```

Comment:
Fett med createReducer 🎉 ble litt gira så her er et lite forslag for enda mer generalisering: 
I stedet for å ta inn alle typene, ta inn navnet på reduceren så kan du bare concate på `_FEILET`, `_OK`, osv